examples/test-notification.php

```diff
@@ -14,7 +14,7 @@
 $clientSecret = $_ENV['CLIENT_SECRET'];
 
 $parameters = [
-    'id'   => '3081704379',
+    'id'   => '3081796354',
```

Comment:
What about using some `ENV` variable? This is changed everytime with new testing ...

Reply:
yes i don't like it also but don't want to change repo style ;-) co i continued with the conventions. but this could be definitely changed in future. not sure if it is related to this MR